llvm-external-projects/iree-dialects/lib/Dialect/LinalgExt/Transforms/Transforms.cpp

```diff
@@ -784,6 +792,148 @@ createLinalgStrategyRemoveMarkersPass() {
   return std::make_unique<LinalgStrategyRemoveMarkersPass>();
 }
 
+//===----------------------------------------------------------------------===//
+// LinalgExt patterns and passes.
+//===----------------------------------------------------------------------===//
+
+namespace {
+template <typename T>
+SmallVector<T> interchange(ArrayRef<T> elements,
```

Comment:
Seems like we have this many places... might be worth moving to a utility function either in `LinalgExt` or in MLIR

Reply:
Yes, I'll do the cleanup afterwards.